.dockerignore

```diff
@@ -1,3 +1,5 @@
 tmp/
 data/
 log/*
+.git
```

Comment:
Not very familiar with `.dockerignore` - does this mean we can alter this line? https://github.com/department-of-veterans-affairs/vets-api/blob/master/Makefile#L22

Some context: these pesky `.git` directories were being left behind by docker, such that, the `.git/refs/tags/` were all owned by `root`. This meant that Jenkins couldn't clean the workspace since it runs as the `jenkins` user.

Reply:
maybe.   the .dockerignore change will prevent the git dir from being copied into the container, but the tests run with a checkout mounted into the container so I don't think that will have much effect.  I'll need to test a bit with jenkins builds to get them working, will try to fix that in the process